triggers/jcdas_dump_post.wc2.pbs

```diff
@@ -31,11 +31,12 @@ PDYm1=$(date -d "${PDY} 1 day ago" +%Y%m%d)
 
 userROOT=/lfs/h2/emc/obsproc/noscrub/$USER
 
-export obsproc_ver=v1.0.0
+export obsproc_ver=v1.0
+export obsproc_ver_pckg=v1.0.0
 export bufr_dump_ver=1.0.0
 export obsNET=obsproc
 
-#export HOMEobsproc=${PACKAGEROOTpara}/obsproc.${obsproc_ver}   # NCO para
+#export HOMEobsproc=${PACKAGEROOTpara}/obsproc.${obsproc_ver_pckg}   # NCO para
 #export HOMEobsproc=${PACKAGEROOT}/${obsNET}.${obsproc_ver}      # NCO prod
```

Comment:
line 39 should use $obsNET (but not imperative)
line 40  needs $obsproc_ver_pckg